guides/common/modules/proc_integrating-idm-dns-with-gss-tsig-authentication.adoc

```diff
@@ -0,0 +1,162 @@
+[id="integrating-idm-dns-update-with-gss-tsig-authentication"]
+= Integrating {FreeIPA} DNS with GSS-TSIG authentication
+
+You can configure the {FreeIPA} server to use the generic security service algorithm for secret key transaction (GSS-TSIG) technology defined in https://tools.ietf.org/html/rfc3645[RFC3645].
+To configure the {FreeIPA} server to use the GSS-TSIG technology, you must install the {FreeIPA} client on the {ProductName} base operating system.
+
+
+.Prerequisites
+
+* The {FreeIPA} server is deployed and functional.
+* The firewall on the {FreeIPA} server allows access to the required ports.
+ifndef::orcharhino[]
+See {RHELDocsBaseURL}/9/html/installing_identity_management/preparing-the-system-for-ipa-server-installation_installing-identity-management#port-requirements-for-idm_preparing-the-system-for-ipa-server-installation[Port requirements for {FreeIPA}] in the _{RHEL}{nbsp}9 Installing Identity Management guide_.
+endif::[]
+* The {FreeIPA} account has permissions to create zones on the {FreeIPA} server.
+
+
+.Procedure
+
+. Create a Kerberos principal on the {FreeIPA} server:
+
+.. Obtain a Kerberos ticket:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# kinit _<{FreeIPA}_user>_
+----
+
+.. Create a new Kerberos principal {ProductName} to use to authenticate on the {FreeIPA} server:
+
+*** For a {ProjectServer}, enter:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# ipa service-add _{smart-proxy-principal}/{foreman-example-com}_
+----
+
+*** For a {SmartProxyServer}, enter:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# ipa service-add _{smartproxy-example-com}_
+----
+
+
+. Install and configure the {FreeIPA} client on either the {Project} or {SmartProxy} that is managing the DNS service for your deployment:
+
+.. Install the `ipa-client` package:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {project-package-install} ipa-client
+----
+
+.. Install the {FreeIPA} client:
++
+[options="nowrap"]
+----
+# ipa-client-install
+----
++
+Follow the on-screen prompts.
+
+.. Obtain a Kerberos ticket:
++
+[options="nowrap"]
+----
+# kinit admin
+----
+
+.. Remove any preexisting `keytab`:
++
+[options="nowrap"]
+----
+# rm --force /etc/foreman-proxy/dns.keytab
+----
+
+.. Obtain a `keytab` file for this system:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# ipa-getkeytab -p {smart-proxy-principal}/_{foreman-example-com}@EXAMPLE.COM_ \
+-s _server.example.com_ -k /etc/foreman-proxy/dns.keytab
```

Comment:
`server.example.com` refers to the FreeIPA server? In that case, we should make that clear. Considering that this will differ across the builds, an attribute might be needed, what do you think? Look at `common/attributes-satellite.adoc` and `common/attributes-base.adoc` and the `:foreman-example-com:` attribute that's defined there. Do you think it makes sense to create something like that for freeipa.example.com and idm.example.com?

Reply:
This would be in line with what we already have for some other branded services. Notably `ovirt-example-com`.

Reply:
I added an attribute.